### ISSUE TITLE
Release v1.59.11

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#8354]
:robot: A dedicated issue to run tests and deploy release [v1.59.11](https://github.com/deckhouse/deckhouse/milestone/310).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe | be | se`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.59.11 | test-v1.59.11*` or a branch `main | release-1.59`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.25 | 1.26 | 1.27 | 1.28 | 1.29 | Automatic`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.59.11 | test-v1.59.11*` or a branch `main | release-1.59`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.25'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/k8s/1.27
/e2e/use/k8s/1.29

This comment will run 2 e2e jobs on AWS with containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/k8s/1.27

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.59
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [c1e41e](https://github.com/deckhouse/deckhouse/commit/c1e41e6e86f3ed14ab6a14a8dd8296c62a76e667) in branch **main**:
```
[deckhouse-controller] refactor controllers with common manager (#8301)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b1b075](https://github.com/deckhouse/deckhouse/commit/b1b0757bcc8df281fa05662bd1a0760e988af8a5) in branch **main**:
```
[docs] Add a linter for spell checking the documentation (#7490)

Signed-off-by: Zhbert <konstantin.nezhbert@flant.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4c1c04](https://github.com/deckhouse/deckhouse/commit/4c1c04dc3829c481913438add38c17d018aa0f23) in branch **main**:
```
[monitoring-applications] Fix rabbitmq dashboard (#7979)

Signed-off-by: DevopsCreator <139634455+DevopsCreator@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3529e2](https://github.com/deckhouse/deckhouse/commit/3529e2256597d5c5c644812d2c66303e4b54ddaf) in branch **main**:
```
[istio] Making the `trafficRedirectionSetupMode` module parameter public (#8353)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
Signed-off-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
Co-authored-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [58b54b](https://github.com/deckhouse/deckhouse/commit/58b54b393c8cfa7c2c519409e30202f186365aea) in branch **main**:
```
Re-generate changelog v1.59.11 (#8357)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: nabokihms <nabokihms@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.59.11** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [1c5ef9](https://github.com/deckhouse/deckhouse/commit/1c5ef9105c8922ca47908ee1a73be9b52bde8b40) in branch **main**:
```
[dhctl] mirror: backport fix for pulling of additional modules images from d8-cli (#8364)

Signed-off-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
Co-authored-by: Maxim Vasilenko <maksim.vasilenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [bb12e5](https://github.com/deckhouse/deckhouse/commit/bb12e57905c93bf1913c37d00d9f8d414253951a) in branch **main**:
```
[istio] Optimizing matrix tests (#8363)

Signed-off-by: Andrey Polovov <andrey.polovov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [1b4083](https://github.com/deckhouse/deckhouse/commit/1b40835c08f0abb94ea8ebd2b029ce9234d12aae) in branch **main**:
```
Re-generate changelog v1.59.11 (#8366)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.59.11** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [a99fb2](https://github.com/deckhouse/deckhouse/commit/a99fb2850b9098c15df082d4d1dba61e858b625f) in branch **main**:
```
update faq (#8344)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [55b439](https://github.com/deckhouse/deckhouse/commit/55b4399f16c674eb353447de9b1402875d6eeb37) in branch **main**:
```
[dhctl] feat(dhctl-for-commander): production ready commander mode (part 3) (#7157)


Signed-off-by: Timofey Kirillov <timofey.kirillov@flant.com>
Signed-off-by: Ilya Mayorov <ilya.mayorov@flant.com>
Co-authored-by: Ilya Mayorov <ilya.mayorov@flant.com>
Co-authored-by: Ilya Mayorov <37873799+faroyam@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [397731](https://github.com/deckhouse/deckhouse/commit/3977319f258d91e121439eccd9272bb0f2ef6280) in branch **main**:
```
[dhctl] Add debug log file for all commands (#8303)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2b1111](https://github.com/deckhouse/deckhouse/commit/2b11119505fb7582b6704f2b27370dd41767ecd0) in branch **main**:
```
[docs] Fix spelling in the documentation (#8360)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d80ea2](https://github.com/deckhouse/deckhouse/commit/d80ea28875426b715dd810ec69adfb9b4701b765) in branch **main**:
```
[ci][docs] Skip building documentation on some cases (#8379)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```
